### PR TITLE
fixes styling in grade-school

### LINKED ISSUE #399: Fix code styles of all example solutions
~~**Blocked by #373**~~

---

We already have ~~an issue to add~~ added linting capabilities to our codebase using eslint. Now we need to make all exercise files compliant with the linting rules. 

We will be spitting it up into exercises and would like to have a separate PR for each exercise.

**Ready to go:**
Below is a list of all files that need linting along with the ones which are already linted (checked).
See the full list of exercises here: https://github.com/exercism/javascript/tree/master/exercises

- [x] accumulate
- [x] acronym
- [x] all-your-base
- [x] allergies
- [x] alphametics
- [x] anagram
- [x] atbash-cipher
- [x] beer-song
- [x] binary
- [x] binary-search
- [x] binary-search-tree
- [x] bob
- [x] bowling
- [x] bracket-push
- [x] change
- [x] circular-buffer
- [x] clock
- [x] collatz-conjecture
- [x] connect
- [x] crypto-square
- [x] custom-set
- [x] diamond
- [x] difference-of-squares
- [x] diffie-hellman
- [x] etl
- [x] flatten-array
- [x] food-chain
- [x] forth
- [x] gigasecond
- [x] grade-school
- [x] grains
- [x] hamming
- [x] hello-world
- [x] hexadecimal
- [x] isbn-verifier
- [x] isogram
- [x] kindergarten-garden
- [x] largest-series-product
- [x] leap
- [x] linked-list
- [x] list-ops
- [x] luhn
- [x] matrix
- [x] meetup
- [x] minesweeper
- [x] nth-prime
- [x] nucleotide-count
- [x] ocr-numbers
- [x] octal
- [x] palindrome-products
- [x] pangram
- [x] pascals-triangle
- [x] perfect-numbers
- [x] phone-number
- [x] pig-latin
- [x] point-mutations
- [x] prime-factors
- [x] protein-translation
- [x] proverb
- [x] pythagorean-triplet
- [x] queen-attack
- [x] raindrops
- [x] reverse-string
- [x] rna-transcription
- [x] robot-name
- [ ] robot-simulator
- [x] roman-numerals
- [x] run-length-encoding
- [x] saddle-points
- [x] say
- [x] scrabble-score
- [x] secret-handshake
- [x] series
- [x] sieve
- [ ] simple-cipher
- [x] simple-linked-list
- [x] space-age
- [x] strain
- [x] sublist
- [x] sum-of-multiples
- [x] transpose
- [x] triangle
- [x] trinary
- [x] twelve-days
- [x] two-bucket
- [x] two-fer
- [x] word-count
- [x] wordy
- [x] zipper

Our CI will check against the linting rules, but since they are not yet implemented for exercises, the [respective directories have been ignored](https://github.com/exercism/javascript/blob/master/.eslintignore).

**Task**
1. Pick one exercise from list of exercises - see if it is not already implemented in the above list (it should not be checked).
2. Comment here that you are working on that exercise.
3. Remove the exercise line from [`.eslintignore`](https://github.com/exercism/javascript/blob/master/.eslintignore)
4. Run `npm run lint` and see all linting errors. 
5. Some of these are auto-fixable using `npm run lint-fix`., others have to be fixed manually.
6. Commit and submit a PR 🎉 

**Note:** Please do not fix `big-integer.js` and `big-integer.spec.js` since those are third party code (for compatibility reasons) and we don't want to touch that. Let it remain in `eslintignore`.

### PR DESCRIPTION
**Pull request by SamDesai333**
_removed "fixes" language_ -MM

this PR fixes the variable "grade" in exercise grade-school which was previously declared in the upper scope.

### CODE REVIEW

**Comment by matthewmorgan** [comment]
Hi @SamDesai333 thanks for your contribution!

Two things:
1. There are merge conflicts with your branch now that earlier PRs have been merged in.  Can you please merge in master and push your changes to this PR?  Please let me know if you need help.
2. Since #399 applies to multiple apps, please don't reference 'Fixes' or 'Closes' in your description-- GitHub will close that issue when it's merged!

Thanks!

File: exercises/grade-school/example.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 module.exports = function School() {
   var db = {};
 
-  function add(student, grade) {
-    if (db[grade]) {
-      db[grade].push(student);
+  function add(student, gradeLvl) {
```
Comment (matthewmorgan) [inline exercises/grade-school/example.js]: Since we're making this change, let's make it totally readable: `gradeLevel` please!

**Comment by matthewmorgan** [comment]
Thanks @SamDesai333 !!

**Comment by SamDesai333** [comment]
Yeah no problem!

**Pull request merged**